tools/build_variables.py

```diff
@@ -83,7 +83,9 @@
     "torch/csrc/jit/passes/remove_expands.cpp",
     "torch/csrc/jit/passes/requires_grad_analysis.cpp",
     "torch/csrc/jit/passes/shape_analysis.cpp",
-    "torch/csrc/jit/passes/specialize_autogradzero.cpp",
+    "torch/csrc/jit/passes/specialize_undef.cpp",
```

Comment:
This addition looks strange. Could you please double-check that it's not some merge artifact?